pangeo-notebook/conda-lock.yml

```diff
@@ -30582,11 +31092,11 @@ package:
     xarray: '>=0.10.8'
     matplotlib-base: '>=2.0.2'
     netcdf4: '>=1.1.9'
-    zarr: '>=2.11.0,!=2.18.0'
-  url: https://conda.anaconda.org/conda-forge/noarch/parcels-3.1.1-pyhea20964_1.conda
+    zarr: '>=2.11.0,!=2.18.0,<3'
```

Comment:
Oo, `parcels` has pinned to `zarr<3` at https://github.com/conda-forge/parcels-feedstock/pull/129, need to keep an eye on https://github.com/OceanParcels/Parcels/issues/1819